A66-otel-stats.md

```diff
@@ -0,0 +1,305 @@
+OpenTelemetry plugin. Overall, the APIs should have the following capabilities -
+
+*   Allow installing multiple OpenTelemetry plugins.
+*   Allow setting a
+    [MeterProvider](https://opentelemetry.io/docs/specs/otel/metrics/api/#meterprovider)
+    on individual plugins.
+*   Optionally allow enabling/disabling metrics. This would allow optimizations
+    to avoid computation and collection of expensive stats within the gRPC
+    library. Note that even without this capability, users of OpenTelemetry
+    would be able to customize
+    [views](https://opentelemetry.io/docs/specs/otel/metrics/sdk/#view) through
+    the MeterProvider.
+*   Optionally allow setting of a OpenTelemetry plugin for a specific channel or
+    server, instead of setting it globally.
+
+Note that implementations of the gRPC OpenTelemetry plugin
+[should prefer](https://opentelemetry.io/docs/specs/otel/overview/) to only
+depend on the OpenTelemetry API and not the OpenTelemetry SDK.
+
+### C++
+
+```c++
+Class OpenTelemetryPluginBuilder {
+ public:
+  // Enables base set of metrics by default
+  OpenTelemetryPluginBuilder() = default;
+  // If `SetMeterProvider` is called, \a meter_provider will be used instead of the global default.
+  OpenTelemetryPluginBuilder& SetMeterProvider(shared_ptr<opentelemetry::metrics::MeterProvider> meter_provider);
+  // Enable metric \a metric_name.
+  OpenTelemetryPluginBuilder&  EnableMetric(absl::string_view metric_name);
+  // Disable metric \a metric_name
+  OpenTelemetryPluginBuilder&  DisableMetric(absl::string_view metric_name);
+  // Builds and registers a OpenTelemetry Plugin
+  void BuildAndRegisterGlobal();
+};
+
+```
+
+In the future, additional API might be provided to allow registering the plugin
+for a particular channel or server builder.
+
+### Java
+
+To be filled
+
+### Go
+
+To be filled
+
+### Python
+
+To be filled
+
+## Migration from OpenCensus
+
+The following sections show the differences between the gRPC OpenCensus spec and
+the proposed gRPC OpenTelemetry spec and the mapping of metrics between the two.
+It also presents metrics present in OpenCensus spec that do not map to a metric
+in the OpenTelemetry spec at present. Two migration strategies are also proposed
+for customers who are satisfied with the stats coverage provided by the current
+OpenTelemetry spec.
+
+### Differences from gRPC OpenCensus Spec
+
+*   OpenTelemetry instrument names don’t allow ‘/’ so we use ‘.’ as the
+    separator. We also get rid of the “.io” suffix in “grpc.io” as it doesn’t
+    seem to add any value and is consistent with other names in the metrics spec
+    from OpenTelemetry.
+*   We also use this opportunity to resolve ambiguities from the gRPC OpenCensus
+    spec (detailed below).
+*   OpenTelemetry has attributes similar to tags in OpenCensus, and the
+    OpenCensus tag names already seem to match the OpenTelemetry spec - except
+    for ‘_’ vs ‘.’ for namespaces. So we just replace the ‘_’ with ‘.’. Note
```

Comment:
```suggestion
    for ‘\_’ vs ‘.’ for namespaces. So we just replace the ‘\_’ with ‘.’. Note
```

Underscores are markdown formatting characters so they need to be escaped when used like this.